bridge/interface.go

```diff
@@ -31,3 +38,13 @@ type Bridge interface {
 	Execute(context.Context, ActionId, *Batch) (string, error)
 	SignersCount(context.Context, ActionId) uint
 }
+
+// ElrondProxy defines the behavior of a proxy able to serve Elrond blockchain requests
+type ElrondProxy interface {
```

Comment:
My thinking initialy with this package was to only have the interfaces that will be shared with multiple packages, `ElrondProxy` it's only used in `elrond/client`

Reply:
will be used elsewhere and most likely will be moved